### PR TITLE
Lesson05

### PR DESCRIPTION
**Pull request by aimzOG**
assignment 05 as a draft for final project site.

### CODE REVIEW

File: lesson05/assignment/index.html [hunk 1]
```diff
@@ -1,14 +1,59 @@
...
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="css/style.css" />
+    <title>Assignment 05 - CSS Grid</title>
+</head>
+
+<body>
+    <div class="container">
+        <header>
+            <h1 class="heading">Heading</h1>
+            <nav class="nav">
```
Comment (brycebenson) [inline lesson05/assignment/index.html:16]: For the BEM requirement, you can just update the nav component and its children to utilize some sort of BEM classes.

File: lesson05/assignment/index.html [hunk 2]
```diff
@@ -1,14 +1,59 @@
...
+                </ul>
+            </nav>
+        </header>
+
+        <!-- ingredients and recipe -->
         <main class="main">
-          
+            <h2>Main Content</h2>
+            <p>banana banoonoo ban banibi. banana banoonoo ban banibi. banana banoonoo
+                ban banibi. banana banoonoo ban banibi. banana banoonoo ban banibi.
```
Comment (brycebenson) [inline lesson05/assignment/index.html:31]: Love this song.

File: lesson05/assignment/css/_components.content.scss [hunk 3]
```diff
@@ -8,6 +8,90 @@ html {
...
+}
+
+//grid template for mobile
+.container {
+    background-image: url(../img/cutting-board-background-2065305_1920.jpg);
+    background-size: cover;
+    display: grid;
+    grid-template-columns: auto;
+    grid-template-rows: auto;
+    grid-template-areas: 'header'
```
Comment (brycebenson) [inline lesson05/assignment/css/_components.content.scss:22]: Good use of grid features

**Review by brycebenson — CHANGES_REQUESTED** [review]
Great responsive work here. For the last point, just update some elements in your HTML/CSS to utilize some BEM methodology.

File: lesson05/assignment/index.html (hunk 1 above)
Comment (aimzOG) [reply lesson05/assignment/index.html:16]: Updated!

**Review by brycebenson — APPROVED** [review]
Looks like the class="" got left off the li and a updates, but you got the idea.

**Pull request merged**